### DISCUSSION TITLE
Acerca de facturas simplificadas pero con información del destinatario

### DISCUSSION THREAD

**Discussion by ortegafernando** [Barnetik/tbai-php-lib#46] [Q&A] [answered]
Hola, relacionado con la pregunta frecuente nº44 de BATUZ (Bizkaia), ¿es posible aquí gestionar una factura que si se marque como simplificada pero que incluya la información del destinatario?

Gracias.

_Nº44 10/12/2020
¿Se puede expedir con TicketBAI una factura simplificada en la que se identifique el destinatario de la operación?
Sí. Se trata de las llamadas "facturas simplificadas cualificadas" a que se refieren los apartados 2 y 3 del artículo 7 del Reglamento de facturación.
En este caso, la factura simplificada cualificada se identificará con la clave "S" en el campo "Factura simplificada" y además se deberán recoger los datos identificativos del destinatario de la operación (por lo menos el NIF del mismo o, en su caso, otro número identificativo)_

**Comment by ortegafernando** [comment]
Kaixo, alguien sabe sobre esto?

**Comment by arraintxo** [comment]
Kaixo @ortegafernando ,

en principio no debería haber problema. Si estás usando el createFromJson para generar las facturas, basta con que añadas la sección de recipients para que estos datos se añadan a la factura.

Por ejemplo:
```
{
  "territory": "01",
  "subject": {
    "issuer": {
       ...
    },
    "recipients": [
      {
        "vatId": "00000000T",
        "vatIdType": "02",
        "name": "Client Name",
        "postalCode": "48270",
        "address": "Markina-Xemein",
        "countryCode": "ES"
      }
    ],
    "issuedBy": "N"
  },
  "invoice": {
    "header": {
      "series": "TESTSERIE",
      "invoiceNumber": "1646674248",
      "expeditionDate": "07-03-2022",
      "expeditionTime": "17:30:48",
      "simplifiedInvoice": true
    },
   "data": {
    ....
   },
  "breakdown": {
   ...
  }

}
```

Un saludo,

**Comment by ortegafernando** [comment]
Gracias pero no lo estoy creando vía json, lo estoy creando via funciones y clasess Issuer, recipient, Subject, etc....  No soy experto, quizás se pueda conseguir también?

**Comment by arraintxo** [comment, marked as answer]
Sin problema, puedes ver como se hace en los tests: [TicketBaiMother::createSimplifiedTicketBai](https://github.com/Barnetik/tbai-php-lib/blob/4f850a2b3b5d625cffcfa758e1bc6918d4135fff/tests/Barnetik/Tbai/Mother/TicketBaiMother.php#L98)

Básicamente es generar una factura simplificada y añadir un Subject normal con Recipient.
